src/index.js

```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react'
 import ReactDOM from 'react-dom'
 import { Ocean } from '@oceanprotocol/squid'
 import Web3 from 'web3'
-import asset from './asset'
+import { asset } from './asset'
```

Comment:
we should use only one way of importing/exporting, it is a common source of confusion for newcomers which this tutorial is targeted for. Using a default export for `asset` as before would keep it consistent with the rest